Bottom: 1b58f101752654c4e003a6aaa9eb94ddad8314f7
Top:    7918b1fe7705b7bb80a13a584756ecfbe4551f49
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 00:02:58 +0000

Refresh of dangling-fixup-from-old-way-of

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 21f8211..4636319 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -330,14 +330,14 @@ object3d_from_board_outline (void)
         {
           faces[i] = make_face3d ();
 
-            object3d_add_face (object, faces[i]);
-            /* Pick one of the upright edges which is within this face outer contour loop, and link it to the face */
+          object3d_add_face (object, faces[i]);
+          /* Pick one of the upright edges which is within this face outer contour loop, and link it to the face */
 #ifdef REVERSED_PCB_CONTOURS
-            face3d_add_contour (faces[i], make_contour3d (edges[2 * npoints + i]));
+          face3d_add_contour (faces[i], make_contour3d (edges[2 * npoints + i]));
 #else
-            face3d_add_contour (faces[i], make_contour3d (SYM(edges[2 * npoints + i])));
+          face3d_add_contour (faces[i], make_contour3d (SYM(edges[2 * npoints + i])));
 #endif
-          }
+        }
 
       faces[npoints] = make_face3d (); /* bottom_face */
 #ifdef REVERSED_PCB_CONTOURS
